tensorflow/contrib/crf/python/kernel_tests/crf_test.py

```diff
@@ -199,6 +200,87 @@ def testViterbiDecode(self):
       self.assertEqual(actual_max_sequence,
                        expected_max_sequence[:sequence_lengths])
 
+  def testCrfDecode(self):
+
+    def np_crf_decode(potentials, transition_params, seq_lens):
+      '''This is a function for numpy arrays.
```

Comment:
Use double quotes for consistency.

Here and  below, please backtick symbols, e..g `[B, T, num_tags]`.